interface/forms/dictation/common.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Dictation form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    cfapress <cfapress>
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Robert Down <robertdown@live.com>
+ * @copyright Copyright (c) 2008 cfapress <cfapress>
+ * @copyright Copyright (c) 2013-2017 bradymiller <bradymiller@users.sourceforge.net>
+ * @copyright Copyright (c) 2017-2023 Robert Down <robertdown@live.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ **/
+
+require_once(__DIR__ . "/../../globals.php");
+require_once("$srcdir/api.inc.php");
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Core\Header;
+
+$returnurl = 'encounter_top.php';
+
+?>
+<html>
+<head>
+    <title><?php echo xlt("Dictation"); ?></title>
+    <?php Header::setupHeader(['ckeditor', 'angular', 'angular-sanitize']);?>
+</head>
+<body>
+<div class="container mt-3">
+    <div class="row">
+        <div class="col-12">
+            <h2><?php echo xlt("Dictation"); ?></h2>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-12" id="toptext">
+            <form name="my_form" method=post action="<?php echo $rootdir;?>/forms/dictation/save.php?mode=new" onsubmit="return top.restoreSession()">
```

Comment:
hi @juggernautsei , the save script needs to differentiate on mode=new or mode=update along with a passed in id param of the form right?

Reply:
Yes, that is correct.